.github/workflows/packaging-test-pipelines.yml

```diff
@@ -141,9 +151,21 @@ jobs:
           make clean
 
       - name: Make
+        shell: bash
         run: |
+          git config --global --add safe.directory ${GITHUB_WORKSPACE}
           make -sj$(cat /proc/cpuinfo | grep "core id" | wc -l) 2>&1 | tee -a output.log
```

Comment:
Cant we just run `set -e` before this line so that pipe failures would cause the execution exit with failure code immediately?

See reference: https://www.gnu.org/software/bash/manual/html_node/The-Set-Builtin.html#:~:text=%2De,single%20simple%20command

Reply:
Sure